src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { createStore, combineReducers } from 'redux';
```

Comment:
- [OPTIONAL] You import  `applyMiddleware`  to use some handy `Redux middleware`, like `redux logger` that will help you a lot in the development process. To install it, run `npm i --save redux-logger`
 in the root directory of your project.

Reply:
alright I'll do it